src/tribol/mesh/CouplingScheme.cpp

```diff
@@ -403,13 +406,6 @@ bool CouplingScheme::isValidCouplingScheme()
    {
       return false;
    }
-   
-   // set boolean for null meshes
-   if ( mesh1.m_numCells <= 0 || mesh2.m_numCells <= 0 )
-   {
-      this->m_nullMeshes = true;
-      valid = true; // a null-mesh coupling scheme should still be valid
```

Comment:
Just to be thorough, I see that we no longer need to set valid = true because valid is initialized to true, and stays true when we set the boolean for null meshes in the new lines of code.